### PR TITLE
Introduce toasts and use them in the checks settings

### PR DESCRIPTION
**Pull request by dottorblaster**
We still didn't have a nice way to fire toasts inside our React applications, so I did some research and I found `react-toastify`, a library that had three requirements I wanted for our solution:

- Allows to fire toasts with a single function call
- Doesn't pollute our React apps with `<NotificationProvider />` or similar weird containers
- Allows us to render custom JSX components, put simply, it allows us to render EOS toasts compliant with our brand and design system.

With some tinkering, I discovered that Toastify has a "lazy container mode", so we can configure it to mount a container automatically when it needs so, and I made a Toast component that we will reuse from now on.

Then, I finally plugged this inside our settings modal.

Peek 13-09-2021 14-52

### CODE REVIEW

**Review by nelsonkopliku — APPROVED** [review]
Well deserved LGTM! :joy:

File: web/frontend/components/Toast/Toast.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
+import React from 'react';
+import { toast } from 'react-toastify';
+import Toast from 'react-bootstrap/Toast';
+
+import 'react-toastify/dist/ReactToastify.min.css';
+
+toast.configure({ hideProgressBar: true, closeButton: false });
+
+export const showSuccessToast = ({ title = 'Success!', content = '' }) =>
+  toast(({ closeToast }) => (
```
[the remark this answers is not in the document]
Comment (dottorblaster) [reply web/frontend/components/Toast/Toast.js:10]: Exactly! The internals of `react-toastify` will do something like

```
export const toast = (renderFunction) => {
  const closeToast = () => {
    // IDK, close toast :D
  };
  renderFunction({ closeToast });
};
```

[the remark this answers is not in the document]
Comment (nelsonkopliku) [reply web/frontend/components/Toast/Toast.js:10]: thank you master :wink:

File: web/frontend/components/Toast/Toast.js [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+          className="rounded mr-2 toast-logo"
+          alt=""
+        />
+        <strong className="mr-auto">{title}</strong>
+      </Toast.Header>
+      <Toast.Body className="trento-toast-body">{content}</Toast.Body>
+    </Toast>
+  ));
+
+export const showErrorToast = ({ title = 'OH NOES', content = '' }) =>
```
Comment (fabriziosestito) [inline web/frontend/components/Toast/Toast.js:24]: LOL

**Comment by fabriziosestito** [comment]
This looks neat, thanks. I was looking to the component library to understand how it works, and it is superclear.

**Review by fabriziosestito — APPROVED** [review]

**Comment by rtorrero** [comment]
Great job 👍 ! LGTM

**Review by rtorrero — APPROVED** [review]

**Pull request merged**